### DISCUSSION TITLE
Pricing after the beta

### DISCUSSION THREAD

**Discussion by nehalist** [community/community#25839] [Actions]
<p>Are there any information out there yet regarding pricing for private repositories after the beta? Right now it states “Free during beta” - but what happens afterwards?</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>Private repo pricing is shown towards the bottom of this page: <a href="https://github.com/features/actions" rel="nofollow noopener">https://github.com/features/actions</a></p>

**Comment by jfhenon** [comment]
<p>Hello, I may have bad eyes but I can’t see the price of each plan (pro, team, …) ? I can however see the additionnal hours for each operating systems. Can you help?</p>
<p>Thanks</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/jfhenon">@jfhenon</a>!</p>
<p>You can find the pricing for Pro and Team on the <a href="https://github.com/pricing" rel="nofollow noopener">GitHub Pricing page</a>. Each one of those plans comes with a certain number of hours with additional hours available at the prices listed on the Acions page. </p>
<p>Hope that helps!</p>

**Comment by jfhenon** [comment]
<p>Yes, it does. I thought it was separate plans for github actions and github. thanks ! </p>

**Comment by hakusaro** [comment]
<p>Is the included minute count per repo, per org, or per person (if a private account upgrades)?</p>

**Comment by mscoutermarsh** [comment]
<p>The included minutes are per User/Org (whichever account type is the owner of the repository where your Actions are running).</p>
<p>For example, an Organization on the Team plan, would have 10,000 minutes included in their plan. Those minutes are shared across all of their repositories.</p>
